### ISSUE TITLE
Require Moodle 3.7 

### ISSUE THREAD

**Issue by leoauri** [mudrd8mz/moodle-tool_pluginskel#92]
Hi there,
I'm just using your plugin skeleton generator to generate a plugin for moodle 3.7.  As I see 
```
!!! An error has occurred during this process !!!
!! Unknown required Moodle version: 3.7
```

I will use 3.6 for now.  What would have to be updated for 3.7 compatibility?  Can I help?

Looks very helpful, thanks
Leo

**Comment by mudrd8mz** [comment]
Can you provide the YAML file with the configuration?

**Comment by leoauri** [comment]
This is enough to get the error:
```
component: mod_fail
requires: "3.7"
```

**Comment by leoauri** [comment]
At ce48bb85af2d37928de32a29045a2112bc718268

**Comment by mudrd8mz** [comment]
Oh right. "Someone" had forgotten to update the plugin when 3.7 was released ;-) The new version 1.2.2 should have it fixed.

**Comment by leoauri** [comment]
Great!  Thanks!
